### DISCUSSION TITLE
How to use in an existing Nuxt3 project?

### DISCUSSION THREAD

**Discussion by MangMax** [nuxt-content/docus#947] [Q&A]
As you can see, I have installed @nuxt/content and @nuxt-themes/docus
![image](https://github.com/nuxt-themes/docus/assets/68525017/56cf33e9-6465-4c22-b8a0-0d8774e2070e)
And the page was created according to the rules.
![image](https://github.com/nuxt-themes/docus/assets/68525017/61339f06-6445-4b00-b661-bbe7096d1aba)
![image](https://github.com/nuxt-themes/docus/assets/68525017/a2a0636f-a0a2-4c83-8cb5-960c7300e130)
![image](https://github.com/nuxt-themes/docus/assets/68525017/94196b4c-ecc1-4ffd-8e45-f67dfc94d4d4)
When I visit http://localhost:3000/examples/test, the expected page is not returned.
![image](https://github.com/nuxt-themes/docus/assets/68525017/6fa460b6-40a6-4116-85f8-603b7aaaaaa8)
May I ask where the problem lies with my configuration?

**Comment by mynameisjeffe** [comment]
As per this [issue](https://github.com/nuxt-themes/docus/issues/966), maybe it's difficult to do

In my case, I was able to get it to work but I didn't need to install @nuxt/content.

Although I had this [issue](https://github.com/nuxt-themes/docus/issues/972) appeared.. So eventually I gave up

**Reply by ColtHands** [reply]
What doc engine you ended up using instead?

**Comment by ColtHands** [comment]
Also + to the comment, however it looks like OP is confused about naming I have a problem where there's no layout rendering.

Would really appreciate a guide on adding into existing Nuxt project, where not all stuff from `/content` is for documentation
